[2025-09-21T11:12:07.206+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:11:00+00:00/task_id=trigger_model_training permission to 509
[2025-09-21T11:12:07.207+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:11:00+00:00/task_id=trigger_model_training permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:11:00+00:00/task_id=trigger_model_training'
[2025-09-21T11:12:07.271+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:11:00+00:00/task_id=trigger_model_training permission to 509
[2025-09-21T11:12:07.273+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:11:00+00:00/task_id=trigger_model_training permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:11:00+00:00/task_id=trigger_model_training'
[2025-09-21T11:12:07.357+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: train_predict_to_db.trigger_model_training scheduled__2025-09-21T11:11:00+00:00 [queued]>
[2025-09-21T11:12:07.370+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: train_predict_to_db.trigger_model_training scheduled__2025-09-21T11:11:00+00:00 [queued]>
[2025-09-21T11:12:07.370+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-09-21T11:12:07.392+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): trigger_model_training> on 2025-09-21 11:11:00+00:00
[2025-09-21T11:12:07.405+0000] {standard_task_runner.py:57} INFO - Started process 419 to run task
[2025-09-21T11:12:07.412+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'train_predict_to_db', 'trigger_model_training', 'scheduled__2025-09-21T11:11:00+00:00', '--job-id', '22', '--raw', '--subdir', 'DAGS_FOLDER/train_predict_to_db_dag.py', '--cfg-path', '/tmp/tmpnpvjfgj4']
[2025-09-21T11:12:07.427+0000] {standard_task_runner.py:85} INFO - Job 22: Subtask trigger_model_training
[2025-09-21T11:12:07.529+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:11:00+00:00/task_id=trigger_model_training permission to 509
[2025-09-21T11:12:07.530+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:11:00+00:00/task_id=trigger_model_training permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T11:11:00+00:00/task_id=trigger_model_training'
[2025-09-21T11:12:07.532+0000] {task_command.py:416} INFO - Running <TaskInstance: train_predict_to_db.trigger_model_training scheduled__2025-09-21T11:11:00+00:00 [running]> on host 36da5371e73d
[2025-09-21T11:12:07.680+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='train_predict_to_db' AIRFLOW_CTX_TASK_ID='trigger_model_training' AIRFLOW_CTX_EXECUTION_DATE='2025-09-21T11:11:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-21T11:11:00+00:00'
[2025-09-21T11:12:13.115+0000] {python.py:194} INFO - Done. Returned value was: 1c01dc3c92454cb58ece1995f027f6f2
[2025-09-21T11:12:13.154+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=train_predict_to_db, task_id=trigger_model_training, execution_date=20250921T111100, start_date=20250921T111207, end_date=20250921T111213
[2025-09-21T11:12:13.219+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-09-21T11:12:13.243+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
